build.gradle

```diff
@@ -26,8 +26,8 @@ buildscript {
     classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.4'
     classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0'
     classpath 'com.android.tools.build:gradle:1.5.0'
-    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
-//    classpath "com.athaydes.gradle.ceylon:ceylon-gradle-plugin:1.1"
+//    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
```

Comment:
@FroMage I think that to solve this problem, you can make `sample-app` depend on the actual jar file created when you build `ceylon-gradle-android`... That's what I did in the ceylon-gradle-plugin test projects, see [this](https://github.com/renatoathaydes/ceylon-gradle-plugin/blob/master/ceylon-gradle-plugin-tests/build.gradle#L5).

This way, you just need to do `gradle jar` on the main project, before you run the sample project.